supportedBackends/google/pipelines/v2beta/src/main/scala/cromwell/backend/google/pipelines/v2beta/api/request/GetRequestHandler.scala

```diff
@@ -115,9 +115,30 @@ trait GetRequestHandler { this: RequestHandler =>
           if (lastDashIndex != -1) zoneString.substring(0, lastDashIndex) else zoneString
         }
 
+        val gpuInfo: Option[GpuInfo] = for {
+          pipelineValue <- pipeline
+          resources <- Option(pipelineValue.getResources)
+          virtualMachine <- Option(resources.getVirtualMachine)
+          gpusList <- Option(virtualMachine.getAccelerators)
+          gpus <- {
+            if (gpusList.size > 1) {
+              // TODO: Improve this warning
+              // - Log appears repeatedly while task is running
+              // - Improve formatting of accelerator info
+              // - Include workflow/task ID?
+              logger.warn(
```

Comment:
There are many reasons why this log message is bad. Please consider but do not limit yourself to the notes in the code comment in the effort to make it good.